src/features/payment/PaymentForm.js

```diff
@@ -206,11 +206,6 @@ const PaymentForm = ({ standalone = true, submitLabel, onDone = () => {} }) => {
   } else {
     return (
       <>
-        {paymentStatus === "unfinished" && (
```

Comment:
I removed this since it's already taken care of by the text at the top, let me know your thoughts